### ISSUE TITLE
Decide how to handle multiple data sources when calculating POIs

### ISSUE THREAD

**Issue by GabrielBruno24** [chairemobilite/transition#1690]
Right now, we calculate the number of POIs in accessibility maps by taking them from all the data sources we have imported in the database, which could be problematic if we have multiple sources covering the same area, as this would double some of the POIs. Maybe we could add a dropdown menu that allows us to select which sources should be included or not?
